cogs/annual_handover_and_reset.py

```diff
@@ -193,7 +192,7 @@ async def annual_roles_reset(self, ctx: TeXBotApplicationContext) -> None:
         for member in member_role.members:
             await member.remove_roles(
                 member_role,
-                reason=f"{ctx.user} used TeX-Bot slash-command: \"/annual_roles_reset\"",
+                reason=f'{ctx.user} used TeX-Bot slash-command: "/annual_roles_reset"',
```

Comment:
Prefer strings to use double quotes & escape interior double quotes rather than using single quotes